diff -rupN ARToolKitPlus-2.1.1/include/ARToolKitPlus/TrackerImpl.h ARToolKitPlus-2.1.1_mod/include/ARToolKitPlus/TrackerImpl.h
--- ARToolKitPlus-2.1.1/include/ARToolKitPlus/TrackerImpl.h	2006-07-25 16:05:46.000000000 +0200
+++ ARToolKitPlus-2.1.1_mod/include/ARToolKitPlus/TrackerImpl.h	2010-05-16 11:53:15.744636401 +0200
@@ -659,40 +659,40 @@ protected:
 
 // this is templated code, so we need to include all this here...
 //
-#include "../../src/extra/FixedPoint.h"
-#include "../../src/core/arBitFieldPattern.cxx"
-#include "../../src/core/arDetectMarker.cxx"
-#include "../../src/core/arDetectMarker2.cxx"
-#include "../../src/core/arGetCode.cxx"
-#include "../../src/core/arGetMarkerInfo.cxx"
-#include "../../src/core/arGetTransMat.cxx"
-#include "../../src/core/arGetTransMat2.cxx"
-#include "../../src/core/arGetTransMat3.cxx"
-#include "../../src/core/rppGetTransMat.cxx" // RPP integration -- [t.pintaric]
-#include "../../src/core/arGetTransMatCont.cxx"
-#include "../../src/core/arLabeling.cxx"
-#include "../../src/core/arMultiActivate.cxx"
-#include "../../src/core/arMultiGetTransMat.cxx"
-#include "../../src/core/rppMultiGetTransMat.cxx" 	// RPP integration -- [t.pintaric]
-#include "../../src/core/arMultiReadConfigFile.cxx"
-#include "../../src/core/arUtil.cxx"
-#include "../../src/core/matrix.cxx"
-#include "../../src/core/mPCA.cxx"
-//#include "../../src/core/paramChangeSize.cxx"
-#include "../../src/core/paramDecomp.cxx"
-#include "../../src/core/paramDistortion.cxx"
-#include "../../src/core/byteSwap.cxx"
-#include "../../src/core/paramFile.cxx"
-#include "../../src/core/vector.cxx"
+#include <ARToolKitPlus/src/extra/FixedPoint.h>
+#include <ARToolKitPlus/src/core/arBitFieldPattern.cxx>
+#include <ARToolKitPlus/src/core/arDetectMarker.cxx>
+#include <ARToolKitPlus/src/core/arDetectMarker2.cxx>
+#include <ARToolKitPlus/src/core/arGetCode.cxx>
+#include <ARToolKitPlus/src/core/arGetMarkerInfo.cxx>
+#include <ARToolKitPlus/src/core/arGetTransMat.cxx>
+#include <ARToolKitPlus/src/core/arGetTransMat2.cxx>
+#include <ARToolKitPlus/src/core/arGetTransMat3.cxx>
+#include <ARToolKitPlus/src/core/rppGetTransMat.cxx> // RPP integration -- [t.pintaric]
+#include <ARToolKitPlus/src/core/arGetTransMatCont.cxx>
+#include <ARToolKitPlus/src/core/arLabeling.cxx>
+#include <ARToolKitPlus/src/core/arMultiActivate.cxx>
+#include <ARToolKitPlus/src/core/arMultiGetTransMat.cxx>
+#include <ARToolKitPlus/src/core/rppMultiGetTransMat.cxx> // RPP integration -- [t.pintaric]
+#include <ARToolKitPlus/src/core/arMultiReadConfigFile.cxx>
+#include <ARToolKitPlus/src/core/arUtil.cxx>
+#include <ARToolKitPlus/src/core/matrix.cxx>
+#include <ARToolKitPlus/src/core/mPCA.cxx>
+//#include <ARToolKitPlus/src/core/paramChangeSize.cxx>
+#include <ARToolKitPlus/src/core/paramDecomp.cxx>
+#include <ARToolKitPlus/src/core/paramDistortion.cxx>
+#include <ARToolKitPlus/src/core/byteSwap.cxx>
+#include <ARToolKitPlus/src/core/paramFile.cxx>
+#include <ARToolKitPlus/src/core/vector.cxx>
 
-#include "../../src/CameraImpl.cxx"
-#include "../../src/CameraAdvImpl.cxx"
-#include "../../src/CameraFactory.cxx"
-#include "../../src/extra/BCH.cxx"
+#include <ARToolKitPlus/src/CameraImpl.cxx>
+#include <ARToolKitPlus/src/CameraAdvImpl.cxx>
+#include <ARToolKitPlus/src/CameraFactory.cxx>
+#include <ARToolKitPlus/src/extra/BCH.cxx>
 
-#include "../../src/TrackerImpl.cxx"
-//#include "../../src/extra/harrisCornerDetector.cxx"
-//#include "../../src/extra/cornerRefinement.cxx"
+#include <ARToolKitPlus/src/TrackerImpl.cxx>
+//#include <ARToolKitPlus/src/extra/harrisCornerDetector.cxx>
+//#include <ARToolKitPlus/src/extra/cornerRefinement.cxx>
 
 
 #endif //__ARTOOLKIT_TRACKERIMPL_HEADERFILE__
diff -rupN ARToolKitPlus-2.1.1/include/ARToolKitPlus/TrackerMultiMarkerImpl.h ARToolKitPlus-2.1.1_mod/include/ARToolKitPlus/TrackerMultiMarkerImpl.h
--- ARToolKitPlus-2.1.1/include/ARToolKitPlus/TrackerMultiMarkerImpl.h	2006-07-25 16:05:46.000000000 +0200
+++ ARToolKitPlus-2.1.1_mod/include/ARToolKitPlus/TrackerMultiMarkerImpl.h	2010-05-16 11:53:15.744636401 +0200
@@ -183,7 +183,7 @@ protected:
 };	// namespace ARToolKitPlus
 
 
-#include "../src/TrackerMultiMarkerImpl.cxx"
+#include <ARToolKitPlus/src/TrackerMultiMarkerImpl.cxx>
 
 
 #endif //__ARTOOLKITPLUS_TRACKERMULTIMARKERIMPL_HEADERFILE__
diff -rupN ARToolKitPlus-2.1.1/include/ARToolKitPlus/TrackerSingleMarkerImpl.h ARToolKitPlus-2.1.1_mod/include/ARToolKitPlus/TrackerSingleMarkerImpl.h
--- ARToolKitPlus-2.1.1/include/ARToolKitPlus/TrackerSingleMarkerImpl.h	2006-07-25 16:05:46.000000000 +0200
+++ ARToolKitPlus-2.1.1_mod/include/ARToolKitPlus/TrackerSingleMarkerImpl.h	2010-05-16 11:53:15.745636170 +0200
@@ -178,7 +178,7 @@ protected:
 
 }	// namespace ARToolKitPlus
 
-#include "../src/TrackerSingleMarkerImpl.cxx"
+#include <ARToolKitPlus/src/TrackerSingleMarkerImpl.cxx>
 
 
 #endif //__ARTOOLKITPLUS_TRACKERSINGLEMARKERIMPL_HEADERFILE__
