diff -rupN ARToolKitPlus-2.1.1/build/linux/options.pro ARToolKitPlus-2.1.1_mod/build/linux/options.pro
--- ARToolKitPlus-2.1.1/build/linux/options.pro	2006-04-19 22:21:08.000000000 +0200
+++ ARToolKitPlus-2.1.1_mod/build/linux/options.pro	2010-05-16 13:26:23.329636259 +0200
@@ -13,8 +13,8 @@ isEmpty(LIBDIR) { LIBDIR = $$PREFIX/lib
 ###
 # choose between debug and release mode
 
-CONFIG		= debug
-#CONFIG		= release
+#CONFIG		= debug
+CONFIG		= release
 
 ###
 # add additional optimization flags (platform-specific)
